packages/api-client/src/api/bundles/bundles.ts

```diff
@@ -99,8 +99,8 @@ export default gql`
                   mediaImages(search: { sort: ASC }) {
                     ...MediaImages
                   }
-                  attributes(filter: $attributeFilters) {
-                    ...Attribute
+                  attributeValues(filter: { isPublic: true, page: $attributesPage, offset: $attributesOffset }) {
+                    ...AttributeValue
```

Comment:
`AttributeValue` fragment should be something like `AttributeValuePaginatedResponse` or whatever it is named in the schema